charts/external-dns/values.yaml

```diff
@@ -229,6 +229,12 @@ domainFilters: []
 ## -- Intentionally exclude domains from being managed.
 excludeDomains: []
 
+## -- (string) Filter resources queried for endpoints by label selector
```

Comment:
```suggestion
# -- (string) Filter resources queried for endpoints by label selector
```

Could you also fix the 2 values above to use a `# -- ` prefix.

Reply:
I see where the mistake came from, thanks. I fix mine and the ones above. The descriptions are now generated